### PR TITLE
EZP-28450: Design improvements for Policies and User and Groups lists

### PR DESCRIPTION
**Pull request by mikadamczyk**
| Question      | Answer
| ------------- | ---
| Tickets       | https://jira.ez.no/browse/EZP-28450
| Bug fix?      |no
| New feature?  | yes
| BC breaks?    |no
| Tests pass?   | yes
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

Design improvements for Policies and User and Groups lists

screen shot 2017-12-08 at 4 39 56 pm

screen shot 2017-12-08 at 4 39 45 pm

#### Checklist:
- [x] Bulk delete in policies
- [x] Bulk delete in user and groups
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/bundle/Resources/views/admin/role/view.html.twig [hunk 1]
```diff
@@ -43,15 +43,13 @@
...
-                {{ 'role.view.view.actions.edit'|trans|desc('Edit') }}
-            </a>
-            {{ form_widget(delete_form.delete, { 'attr': { 'class': 'btn btn-danger' } }) }}
-        </div>
-    </section>
-    {{ form_end(delete_form) }}
+{% block javascripts %}
+    {% javascripts
+    '@EzPlatformAdminUiBundle/Resources/public/js/scripts/admin.policy.list.js'
+    '@EzPlatformAdminUiBundle/Resources/public/js/scripts/admin.role_assignment.list.js'
```
Comment (sunpietro) [inline src/bundle/Resources/views/admin/role/view.html.twig]: Is there a possibility that these files will be included separately into the page layout?
I mean, maybe we should avoid JS code duplication here and create one file with 2 selectors inside?

File: src/bundle/Resources/public/js/scripts/admin.role_assignment.list.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+(function () {
+    const checkboxes = [...document.querySelectorAll('form[name="role_assignments_delete"] input[type="checkbox"]')];
+    const buttonRemove = document.querySelector('#delete-role-assignments');
+    const toggleButtonState = (event) => {
+        const methodName = checkboxes.some(el => el.checked) ? 'removeAttribute' : 'setAttribute';
+        buttonRemove[methodName]('disabled', true);
+    }
+
+    checkboxes.forEach(checkbox => checkbox.addEventListener('change', toggleButtonState, false));
```
Comment (sunpietro) [inline src/bundle/Resources/public/js/scripts/admin.role_assignment.list.js]: The whole code here duplicates the code from `admin.policy.list.js` we can avoid it, by using another selector in the file above.

**Comment by dew326** [comment]
ping @sunpietro

**Comment by lserwatka** [comment]
Rebase is needed here.

**Pull request merged**